Description: Do not use libc for C types, core::ffi is enough
Last-Update: 2023-11-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/build.rs
+++ b/build.rs
@@ -27,2 +27,4 @@
-    #[cfg(not(feature = "std"))]
-    let bindings = bindings.ctypes_prefix("libc");
+    // Debian: core::ffi provides the same c_* types. This also practically
+    // make the std feature a no-op.
+    //#[cfg(not(feature = "std"))]
+    let bindings = bindings.use_core();
